applications/TheLostDiamond_BjornOmsland.md

```diff
@@ -0,0 +1,208 @@
+
+### Overview
+
+- **Total Estimated Duration:** 2 months (for the first milestone)
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 10,000 USD (for the first milestone. The whole project will take two years and the estimated total need for funding is $500,000 USD)
+
+### Milestone 1 — Prove that TheLostDiamond MVP can get a baseline of 100 participants and obtain a 22% monthly growth rate.
+
+- **Estimated duration:** 2 months
+- **FTE:**  1
+- **Costs:** 10,000 USD
+- Milestone 1 is marketing oriented and as such there is no code to test.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | Marketing Campaign No.1 | Diamond No.1 Self-funded. We will add $1,000 dollar to The Lost Diamond and use $500 in marketing. Publish a YouTube-video explaining TheLostDiamond race and how to sign in with Anchor. We will use $1,000 dollar to add 100 x $10 dollar in random checkpoints. Objective: Get monthly active participants. Key result: 10 users pay $7 monthly subscription fee. |
+| 2. | Marketing Campaign No.1 | Diamond No.2 $1,000 is added to The Lost Diamond. $500 dollar is used in marketing. Publish a YouTube-video explaining explaining how EOS earned in the game can be used with a crypto debit card from Coinbase. Get monthly active participants. Key result: 10 users pay $7 monthly subscription fee. |
+| 3. | Marketing Campaign No.1 | Diamond No.3 $1,000 is added to The Lost Diamond. $500 dollar is used in marketing. Publish a YouTube-video about hunting the lost diamond. The third YouTube campaign video will show people hunting for The Lost Diamond. Objective: Get monthly active participants. Key result: 20 users pay $7 monthly subscription fee. |
+| 4. | Marketing Campaign No.2 | Diamond No.4 $1,000 is added to The Lost Diamond. $500 dollar is used in marketing. Publish a YouTube-video about hunting the lost diamond. Objective: Get monthly active participants. Key result: 20 users pay $7 monthly subscription fee. |
+| 5. | Marketing Campaign No.2 | Diamond No.5 $1,000 is added to The Lost Diamond. $500 dollar is used in marketing. Publish a YouTube-video about hunting the lost diamond. Objective: Get monthly active participants. Key result: 20 users pay $7 monthly subscription fee.
+| 6. | Marketing Campaign No.2 | Diamond No.6 $1,000 is added to The Lost Diamond. $500 dollar is used in marketing. Publish a YouTube-video about hunting the lost diamond. Objective: Get monthly active participants. Key result: 20 users pay $7 monthly subscription fee. |  
+| 7. | Marketing Campaign No.2 | Diamond No.7 $1,000 is added to The Lost Diamond. $500 dollar is used in marketing. Publish a YouTube-video about hunting the lost diamond. Objective: Get monthly active participants. Key result: We have a baseline of 100 users paying a $7 monthly subscription fee. |  
+
+### Milestone 2 - Get funding for the next ten $1,000 dollar diamonds
```

Comment:
Similar to the first comment, the milestones should be focused on development costs and not promotional campaigns budgets.

Would be useful to include development tasks defined in Milestone 1 or Milestone 2 (currently not completed)

Reply:
Our problem: We continued on EOS with all the B1-non-funding-struggles and developed a self-funding MVP before the ENF, Helios and Pomelo was available. Then we joined the Helios incubator program, but Helios-investors want to see that our MVP can get traction before investing (and getting investors to a Web3 app where 80% is returned back to the users, is only possible from investors that has an interest in seeing EOS succeed). Then the ENF Grant Framework will only fund development cost (that we have already done ourself) and not promotional campaigns that can give us a chance to prove that the MVP can get traction.

It still feels impossible to get funding for projects on EOS.

The ENF Grant Framework should be focused on helping all projects that has a chance to increase the number of daily active users on the EOS Mainnet. That's what's most important for EOS to succeed.

Please see my Pomelo pitch video: https://youtu.be/arE4cIwBiE8

And please explain why the ENF Grant Framework is focused on development cost only, and not promotional campaigns for an MVP. They should be equal important in the goal of increased DAU on EOS.